scripts/install-windows-prereqs.ps1

```diff
@@ -240,7 +240,7 @@ function Add-ToSystemPath {
         }
     }
     $env:PATH = $currentPath -join ';'
-    setx.exe /M PATH ($systemPath -join ';')
+    setx.exe /M \'PATH\' \'($systemPath -join ';')\'
```

Comment:
To the question you asked me earlier: the `$env:PATH` line sets `PATH` to the scope of the currently executing script, while `setx.exe /M` updates the system-wide `PATH` variable. So I think we need both lines.

However, and especially to get around the max character limit of `setx`, I would instead replace this with in-line C#:

```powershell
[System.Environment]::SetEnvironmentVariable('PATH', $env:PATH, [System.EnvironmentVariableTarget]::Machine)
```

Reply:
FYI @BRMcLaren @johnkord